app/messages/summaries.email.html.erb

```diff
@@ -11,17 +11,19 @@
 <% end %>
 
 <% define_content :footer_link do %>
+<!--
```

Comment:
On Thu, Sep 17, 2015 at 09:19:16AM -0700, Brian Sadler wrote:
> My guess is that this is the link we want to keep and the ```:footer_link``` is the duplicate one that takes you to the root URL.

This thing yields back to the layout and fills the generic one
on the left. It differs in various emails.

I'm keeping the communication profile url which is on the right of
all emails.